src/test/test-cgroup-util.c

```diff
@@ -369,6 +369,17 @@ static void test_is_wanted(void) {
                          "systemd.unified_cgroup_hierarchy=0 "
                          "systemd.legacy_systemd_cgroup_controller=0", 1) >= 0);
         test_is_wanted_print(false);
+
+        /* cgroup_no_v1=all implies systemd.unified_cgroup_hierarchy=1, unless
+         * otherwise explicitly specified. */
+        assert_se(setenv("SYSTEMD_PROC_CMDLINE",
+                         "cgroup_no_v1=all", 1) >= 0);
+        test_is_wanted_print(false);
+
+        assert_se(setenv("SYSTEMD_PROC_CMDLINE",
+                         "cgroup_no_v1=all"
```

Comment:
Hmm, those strings will be concatenated by the compiler, so this test isn't working really.